backend/app/api.py

```diff
@@ -63,6 +63,11 @@ def fetch_jsongz_files():
                 f.write(tv_response.raw.read())
                 print(f"[{tv_path}] downloaded succesfully".replace(directory, ''))
         else:
+            # if we have tried to get data for more than 30 days we give up
+            if day >= 10:
+                print('No downloads for 30 days - giving up')
```

Comment:
10 != 30, or am I misunderstanding something? :D

Reply:
Yeah my bad had to test it with 10 and forgot to change back